Philips.CodeAnalysis.Common/StaticClasses.Allowed.txt

```diff
@@ -0,0 +1 @@
+﻿Philips.CodeAnalysis.Common.Helper
```

Comment:
Ideally we would either make the class not static, or make all the methods extension methods. But this is fine for now.